inference-engine/tests/functional/inference_engine/transformations/split_squeeze_concat_fusion_test.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <string>
+#include <memory>
+
+#include <ngraph/function.hpp>
+#include <ngraph/opsets/opset7.hpp>
+#include <ngraph/pass/manager.hpp>
+#include <transformations/common_optimizations/split_squeeze_concat_fusion.hpp>
+#include <transformations/init_node_info.hpp>
+#include <transformations/utils/utils.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+
+using namespace testing;
+
+TEST(TransformationTests, SplitSqeezeConcatFusion) {
```

Comment:
Please add next tests:
1. (negative) Split has some output not connected to Squeeze operations.
2. (negative) The order of Squeeze ops in Concat is different to (0, 1, 2, 3, ...)

Reply:
Why the 3rd test case is negative?

Reply:
Added tests except for 3rd case, could you please explain what you want from it?